### PR TITLE
Updated the design of the website

### PR DESCRIPTION
**Pull request by SoumyadiptoPal**

### CODE REVIEW

**Comment by AdityaMayukhSom** [comment]
Please write proper description about the commit, add comments in the files, and next time onwards, split  the one large commit into multiple reviewable commits before creating a pull request. 

Refer to [this](https://stackoverflow.com/questions/15772134/can-i-delete-a-git-commit-but-keep-the-changes) to learn how to reset commits without deleting the changes.

File: src/data/themesData.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const themesData = [
+  {
+    themeImageURL: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#7289da" className="w-20 h-20">
```
Comment (AdityaMayukhSom) [inline src/data/themesData.jsx]: Use separate `.svg` extension files for storing the image and use the relative URLs of the images instead. This is due to the fact that any changes to the actual code for the  image is unrelated to the themes data, so separating the inner details of image from actual matter should be done.

Comment (AdityaMayukhSom) [reply src/data/themesData.jsx]: Also after removal of the `<svg>` tag, the file extension should be changed back to `.js` as this does not contain any `jsx` syntax, rather simple data.

File: src/data/prizesData.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  {
+    icon: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#FFD700" className="w-20 h-20">
+    <path fillRule="evenodd" d="M5.166 2.621v.858c-1.035.148-2.059.33-3.071.543a.75.75 0 00-.584.859 6.753 6.753 0 006.138 5.6 6.73 6.73 0 002.743 1.346A6.707 6.707 0 019.279 15H8.54c-1.036 0-1.875.84-1.875 1.875V19.5h-.75a2.25 2.25 0 00-2.25 2.25c0 .414.336.75.75.75h15a.75.75 0 00.75-.75 2.25 2.25 0 00-2.25-2.25h-.75v-2.625c0-1.036-.84-1.875-1.875-1.875h-.739a6.706 6.706 0 01-1.112-3.173 6.73 6.73 0 002.743-1.347 6.753 6.753 0 006.139-5.6.75.75 0 00-.585-.858 47.077 47.077 0 00-3.07-.543V2.62a.75.75 0 00-.658-.744 49.22 49.22 0 00-6.093-.377c-2.063 0-4.096.128-6.093.377a.75.75 0 00-.657.744zm0 2.629c0 1.196.312 2.32.857 3.294A5.266 5.266 0 013.16 5.337a45.6 45.6 0 012.006-.343v.256zm13.5 0v-.256c.674.1 1.343.214 2.006.343a5.265 5.265 0 01-2.863 3.207 6.72 6.72 0 00.857-3.294z" clipRule="evenodd" />
+  </svg>,
+    type: "Overall First",
+    content:
+      "The first overall prize will be given to a project that outstands all other submissions.",
+  },
+  {
+    icon: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#C0C0C0" className="w-20 h-20">
```
Comment (AdityaMayukhSom) [inline src/data/prizesData.jsx]: Please use consistent naming conventions, i.e. as `themesData` was written in camelCase, use camelCase here too. 

The array name can be `prizesData`, same as the name of the file.

File: src/components/themes/Themes.jsx [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-
+import './Themes.css'
 export default function Theme() {
   return (
-    <section class="">
+    <section class="grid">
       {themesData.map((el) => (
-        <div class="">
-          <img src={el.themeImageURL} />
+        <div class="theme_item">
```
Comment (AdityaMayukhSom) [inline src/components/themes/Themes.jsx]: If possible, try to use TailwindCSS for styling instead of the `theme_item` css selector, as it will be easier to maintain later or modify for future websites. Try to use raw CSS as less as possible.

File: src/components/themes/Themes.jsx [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+          <div className="icon">{el.themeImageURL}</div>
           <br />
-          <div class="">{el.themeName}</div>
+          <div class=""><strong style={{fontSize:"larger"}}>{el.themeName}</strong></div>
```
Comment (AdityaMayukhSom) [inline src/components/themes/Themes.jsx:10]: Please setup `prettier` in your preferred text editor and use `Format on save` so that code is automatically formatted before committing. Otherwise on different computers, the code is formatted separately, which creates unnecessary changes in git history.

File: src/components/navbar/Navbar.jsx [hunk 5]
```diff
@@ -45,7 +68,114 @@ const Navbar = () => {
         ))}
       </ul>
     </nav>
-  );
+  );}
+  else
+  {
+    return(
+      <div style={{position:"sticky", top:0, zIndex:100,alignItems:"center", display:"flex",flexDirection:"column"}}>
```
Comment (AdityaMayukhSom) [inline src/components/navbar/Navbar.jsx:75]: Do not use inline styling in JSX, as it is very error prone. Use TailwindCSS instead for utility based CSS system, you will also have intellisense if you are using VS Code with proper extensions.

File: src/components/navbar/Navbar.jsx [hunk 6]
```diff
@@ -20,21 +24,40 @@ const navbarLinks = [
     name: "FAQs",
   },
 ];
-
 const Navbar = () => {
+  const [isNavOpen, setIsNavOpen] = useState(false);
+  const [width,setWidth]=useState(window.innerWidth);
```
Comment (AdityaMayukhSom) [inline src/components/navbar/Navbar.jsx:29]: Do check if `window` object is null or not before using it inside `useState` hook. You can also use a ternary operator for initializing it with some predefined value, otherwise use `setState()` inside `useEffect` for assigning the value after `window` has been initialized.

File: src/components/navbar/Navbar.jsx [hunk 7]
```diff
@@ -20,21 +24,40 @@ const navbarLinks = [
...
+    // Add a resize event listener
+    window.addEventListener('resize', handleResize);
+
+    // Clean up the event listener when the component unmounts
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+  const toggleNav = () => {
+    setIsNavOpen(!isNavOpen);
```
Comment (AdityaMayukhSom) [inline src/components/navbar/Navbar.jsx:44]:
```suggestion
    setIsNavOpen(isNavOpen => !isNavOpen);
```

Refer : https://legacy.reactjs.org/docs/hooks-reference.html#functional-updates

File: src/components/about/About.jsx [hunk 8]
```diff
@@ -1,14 +1,17 @@
...
+       */}
+       <div className="" style={{display:"grid",gridTemplateColumns: "repeat(auto-fit, minmax(400px, 1fr))"}}>
+        <div className="Image" style={{maxWidth: "100%", display:"block", margin:"auto"}}>
+          <img alt="teamwork logo" width='500px' src="/illustrations/TeamWork.svg"
+           />
         </div>
 
-        <div className="">
-          <h1>What is Synchronicity?</h1>
+        <div className="about" style={{display:"block", margin:"auto",maxWidth: "500px", textAlign:"center"}} >
```
Comment (AdityaMayukhSom) [inline src/components/about/About.jsx:12]: Do not use inline styles directly, use TailwindCSS instead.

File: src/components/about/About.jsx [hunk 9]
```diff
@@ -1,14 +1,17 @@
...
     <section className="">
-      <h1>About</h1>
-      <div className="container grid grid-cols-2 p-12 gap-8">
-        <div className="">
-          <img alt="teamwork logo" src="/illustrations/TeamWork.svg" />
+      {/* <div className="container grid grid-cols-2 p-12 gap-8">
+       */}
+       <div className="" style={{display:"grid",gridTemplateColumns: "repeat(auto-fit, minmax(400px, 1fr))"}}>
+        <div className="Image" style={{maxWidth: "100%", display:"block", margin:"auto"}}>
+          <img alt="teamwork logo" width='500px' src="/illustrations/TeamWork.svg"
```
Comment (AdityaMayukhSom) [inline src/components/about/About.jsx:8]: Why is the width attribute in image is a static value?

**Review by AdityaMayukhSom — CHANGES_REQUESTED** [review]
Please go through the points mentioned in the reviews and do the necessary changes, do not close or remove this pull request, after updating, keep pushing on the same branch.

Do not commit all the code at once, commit one group of changes at a time, so that it will be easier for me to review.

File: src/components/Timeline/Timeline.css [hunk 10]
```diff
@@ -99,4 +99,10 @@
 
 .row section p{
   margin: 10px 0 17px 0;
+}
+
+@media (max-width: 710px) {
```
Comment (AdityaMayukhSom) [inline src/components/Timeline/Timeline.css:104]: Use standard viewport sizes, like 320px, 480px, 768px, 1024px, 1200px etc.

**Review by AdityaMayukhSom — CHANGES_REQUESTED** [review]

File: src/components/about/About.jsx (hunk 9 above)
Comment (SoumyadiptoPal) [reply src/components/about/About.jsx:8]: width of the image should be same for both mobile and desktop!?

**Comment by AdityaMayukhSom** [comment]
Merging this PR due to shortage of time before deployment, we can carry on the fixes in future pull requests.

**Review by AdityaMayukhSom — APPROVED** [review]
